xplique/metrics/fidelity.py

```diff
@@ -619,9 +606,97 @@ def __init__(self,
                  targets: Optional[Union[tf.Tensor, np.ndarray]] = None,
                  metric: str = "loss",
                  batch_size: Optional[int] = 64,
-                 baseline_mode: Union[float, str] = 0.0,
+                 baseline_mode: Union[float, Callable] = 0.0,
                  steps: int = 10,
                  max_percentage_perturbed: float = 1.0,
                  ):  # pylint: disable=R0913
         super().__init__(model, inputs, targets, metric, batch_size,
                          "insertion", baseline_mode, steps, max_percentage_perturbed)
+
+
+class DeletionTab(CausalFidelityTS):
+    """
+    Adaptation of the insertion metric for tabular data.
```

Comment:
Insertion or Deletion ? Typo ?

Reply:
oups, I change it ... the class is deletion type